model/file/registry.yaml

```diff
@@ -137,3 +161,18 @@ groups:
           This attribute is only applicable to symbolic links.
         stability: experimental
         examples: ['/usr/bin/python3']
+      - id: file.zone_identifier
```

Comment:
Should this be `file.windows_zone.id`?

Given the generic nature of the word of 'zone' I think this should be clarified.  Additionally we should stick to a similar convention across otel for `id`, and we don't use `identifier` anywhere else that I'm aware of.